.circleci/Makefile

```
@@ -0,0 +1,11 @@
+all default: config.yml
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)